kubernetes/helm/README.md

```diff
@@ -0,0 +1,74 @@
+# Artifactory in Kubernetes using Helm package manager
+This directory has examples for Artifactory in Kubernetes deployments using the [Helm](https://helm.sh/) package manager.
+
+## Helm
+From https://helm.sh/:
+```
+What is Helm?
+
+Helm helps you manage Kubernetes applications — Helm Charts helps you define, install, and upgrade even the most complex Kubernetes application.
+```
+
+Helm in [GitHub](https://github.com/kubernetes/helm)
+
+## Deployment
+
+### Installing helm
+Follow [instructions on installing helm](https://github.com/kubernetes/helm#install)
+ 
+#### Setup Helm to use your cluster
+To initialize Helm and install Tiller, you should run
+```bash
+$ helm init
+```
+
+### Deploying Artifactory Pro
+You can deploy the Artifactory Pro chart, which is in the [artifactory-pro](artifactory-pro) directory
+```bash
+$ helm install -n artifactory-pro ./artifactory-pro
+```
+
+You can package the Artifactory Pro chart and distribute it for use later
```

Comment:
I will move the packaging to the end. Good point.
As for the Artifactory helm support, the link is already in the parent README page.